leakcanary-android-core/src/main/java/leakcanary/internal/HeapAnalyzerService.kt

```diff
@@ -56,20 +58,27 @@ internal class HeapAnalyzerService : ForegroundService(
     val heapAnalyzer = HeapAnalyzer(this)
     val config = LeakCanary.config
 
+    val proguardMapping = try {
+      ProguardMappingReader(assets.open("leakCanaryProguardMapping.txt")).readProguardMapping()
```

Comment:
nits: local variable for ProguardMappingReader and private constant for `"leakCanaryProguardMapping.txt"`